docs/developers/05-programming-patterns/03-ownership.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Ownership
+---
+
+The contract ownership pattern involves designating an account ID as the "owner" of a contract, allowing it to have special permissions. These permissions may include calling certain functions or withdrawing contract funds, among others.
+
+There is not an NEP standard for contract ownership and management.
```

Comment:
@encody "NEP" is lingo. It's confusing. Anywhere it exists in our docs, we should define it and link to what it means. Also, let's explain this sentence. (People will read it and wonder "It's NOT a standard? Does it matter that it's not a standard? What does that mean?")

Reply:
I'll see about adding some sort of glossary / definitions / tooltip feature for this sort of terminology.

Much of the "programming patterns" section is about non-_standard_ but common practice, since only a very limited subset of common practices get standardized (namely, functionality and interface).

Reply:
Good catch @ryancwalsh! I thought about a web3 dictionary/glossary when we started thinking about WT over a month ago but the idea got left behind somehow. We should definitely have such a page or even a section (categorizing the different terms) on our website, and link all the occurrences of those terms to the definition. 

If we don't want to reinvent the wheel, or don't have enough time to define everything ourselves (which I think we should), we can always link those words to an outside source of knowledge (e.g. nomicon or somewhere else).